public/js/index.js

```diff
@@ -2994,7 +2994,8 @@ function initFilterBranchTagDropdown(selector) {
     });
 }
 
-$(".commit-button").click(function() {
+$(".commit-button").click(function(e) {
+    e.preventDefault();
```

Comment:
This isn't needed anymore, right?

Reply:
Still, someone coming across this code will wonder it's purpose. I'd say remove it.

Reply:
To be honest imho it should be for all button/link click events as this prevents from strange behaviors like this bug. Part of click callbacks also have this call without any special purpose mostly, just to prevent bugs like this if layout is broken in some strange resolution or data combination ;)

Reply:
It depends. A button that is not part of a form does to my knowledge never have a default action. A link has of course  a "open" default action which ideally should never be prevented.

Anyways, not a biggie, if you want to keep it, keep it.